src/main/java/racing/domain/RacingCars.java

```diff
@@ -32,11 +41,14 @@ public void startRace() throws CarLocationException {
         }
         carSet.clear();
         carSet.addAll(updatedCarSet);
+        return updatedCarSet;
     }
 
     public Set<Car> getWinCarList() {
         int winnerLocation = getMaxNumberOfList(carSet.stream().map(Car::getLocation).collect(Collectors.toList()));
-        return carSet.stream().filter(car -> car.getLocation() == winnerLocation).collect(Collectors.toSet());
+        return carSet.stream()
+                .filter(car -> car.getLocation() == winnerLocation)
```

Comment:
```suggestion
                .filter(car -> car.이거랑위치같아?(winnerLocation)
```

자동차에게 비교하도록 메시지를 전달해보는건 어때요? 😄